### PR TITLE
Backport dartsass support

### PR DESCRIPTION
**Pull request by januszm**
A combination of a few recent PRs that changed the dependency on SassC to soft and enabled support for dartsass-rails

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Add `bootstrap` to your Gemfile:
 gem 'bootstrap', '~> 4.6.2'
 ```
 
+This gem requires a Sass engine, so make sure you have **one** of these two gems in your Gemfile:
```
Comment (glebm) [inline README.md:23]: Remove "two" because there are 4

Comment (januszm) [reply README.md]: done

**Comment by januszm** [comment]
Tested in one live project and it looks like it finally works, gems install nicely (previously it would fail on Amazon Linux 2023 because sassc gem slow compilation of libsass and/or bad placement of the .so file in the filesystem)

**Comment by glebm** [comment]
Thanks!

**Comment by januszm** [comment]
> Thanks!

Sweet! Please release the 4.6.x gem to rubygems

**Comment by reedstonefood** [comment]
Hi @glebm it's been 6 months - any chance of a rubygems release with this change? 😇

**Comment by glebm** [comment]
Released v4.6.2.1. :tada:

**Pull request merged**